build_modules/build.yaml

```diff
@@ -1,26 +1,51 @@
 builders:
-  modules:
+  module_library:
     import: "package:build_modules/builders.dart"
     builder_factories:
       - moduleLibraryBuilder
-      - metaModuleBuilder
-      - metaModuleCleanBuilder
-      - moduleBuilder
-      - unlinkedSummaryBuilder
-      - linkedSummaryBuilder
     build_extensions:
-      $lib$:
-        - .meta_module.raw
-        - .meta_module.clean
       .dart:
         - .module.library
-        - .module
-        - .linked.sum
-        - .unlinked.sum
-    is_optional: True
     auto_apply: all_packages
+    is_optional: True
     required_inputs: [".dart"]
     applies_builders: ["|module_cleanup"]
+  dartdevc:
+    import: "package:build_modules/builders.dart"
+    builder_factories:
+      - metaModuleBuilderFactoryForPlatform('dartdevc')
```

Comment:
[aside] If we had union types I would totally make the `BuilderFactory` typedef be `Builder|List<Builder> Function()`

Reply:
Yep, would clean this up a bunch